dbos/cli/cli.py

```diff
@@ -252,7 +271,7 @@ def reset(
             raise typer.Exit()
     config = load_config()
     try:
-        reset_system_database(config)
+        start_client().reset_system_database(config)
```

Comment:
Why aren't we doing the new thing here?

Reply:
Ha yes, so https://github.com/dbos-inc/dbos-transact-py/pull/303/commits/d8a26901153524699b4ec58e64cb95a2bd4b467f does one thing I was planning as part of the connection string stuff: make `reset_system_database` simply accept what it needs to work.

The commit above (+ a few other after) gets ride of the config file as a bonus